scrapers/ep_votes/api.py

```diff
@@ -47,75 +47,62 @@ def wrapper() -> Response:
     return wrapper
 
 
-@app.route("/members")
-@json_response
-def members() -> SimpleResponse:
-    term = request.args.get("term", type=int)
+def params(**params: Callable) -> Callable:
```

Comment:
Do you have a link to some documentation or something at hand that broadly explains what this is about?
I see that below we use some flask decorator to define parameters for requests. I also see that this is about checking if all parameters that are needed are given, I assume that flask provides these in `params` then each time. However I do not quite get why/when this wrapper is applied. I think my question boils down to what these SimpleHandlers are, i.e. what constitures SimpleHandlers in our code below. :see_no_evil: